src/mmap_unix.rs

```diff
@@ -8,17 +8,7 @@
 // Use of this source code is governed by a BSD-style license that can be
 // found in the THIRD-PARTY file.
 
-//! A default Unix implementation of the GuestMemory trait by mmap()-ing guest's memory into
-//! the current process.
-//!
-//! The main structs to access guest's memory are:
```

Comment:
Just wondering, why did you delete this part?

Reply:
This info is somehow redundant. It is in `DESIGN.md`. I don't think we have to add it as a header to every file.

Also it seems unrelated with the `mmap_unix.rs` file.